data_buffer.cpp

```diff
@@ -874,16 +877,7 @@ int DataBuffer::get_bit_taken(DataType p_data_type, CompressionLevel p_compressi
 			return get_bit_taken(DATA_TYPE_REAL, p_compression) * 3;
 		} break;
 		case DATA_TYPE_NORMALIZED_VECTOR3: {
-			switch (p_compression) {
-				case CompressionLevel::COMPRESSION_LEVEL_0:
-					return 11 * 3;
-				case CompressionLevel::COMPRESSION_LEVEL_1:
-					return 10 * 3;
-				case CompressionLevel::COMPRESSION_LEVEL_2:
-					return 8 * 3;
-				case CompressionLevel::COMPRESSION_LEVEL_3:
-					return 6 * 3;
-			}
+			return get_bit_taken(DATA_TYPE_UNIT_REAL, p_compression) * 3;
```

Comment:
just making sure the usage of DATA_TYPE_UNIT_REAL is correct here,  (seen previously DATA_TYPE_POSITIVE_UNIT_REAL was used with that func)

Reply:
Yes, it's correct because the vector can have both positive and negative axis. Check the `add_normalized_vector` to see what's doing to confirm we are calculating the size correctly.